cigaradvisor/blocks/article-list/article-list.js

```diff
@@ -29,10 +29,10 @@ export async function renderPage(wrapper, articles, limit) {
     const articleTeaser = document.createElement('div');
     articleTeaser.classList.add('article-teaser');
     articleTeaser.classList.add('block');
-    if (typeof article.author === 'string') {
-      article.author = await fetchAuthorInfo(article.author);
+    if (article.author) {
+      article.author = typeof article.author === 'string' ? await fetchAuthorInfo(article.author) : '';
     }
-    if (typeof article.category === 'string') {
+    if (article.category && typeof article.category === 'string') {
```

Comment:
I'm not sure this is necessary.

`typeof article.category === 'string'` is _false_, when it is undefined, or an object - why introduce a change?